wiki/pages/views.py

```
@@ -1446,10 +1460,45 @@ def confirm_upload(request):
+    check size/type first to avoid pulling a 1 GB video into memory just
+    to skip it. Never raises: the upload already succeeded, and a broken
+    AI call must not turn it into an error.
+    """
+    if upload.content_type not in SUPPORTED_IMAGE_TYPES:
+        return None
+    try:
+        if upload.file.size > MAX_IMAGE_BYTES:
+            return None
+        with upload.file.open("rb") as f:
+            image_bytes = f.read()
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)